src/databricks/labs/ucx/assessment/workflows.py

```diff
@@ -176,6 +176,119 @@ def crawl_groups(self, ctx: RuntimeContext):
         ctx.group_manager.snapshot()
 
 
+class MigrationProgress(Workflow):
```

Comment:
I agree that this is a concern; that said most of these are duplicated from the assessment tasks. (That's not obvious due to the limited context shown in the GH review window.)

The docs have drifted a lot since originally being written. I assumed that the docs were finding their way into the UI somehow, but this doesn't seem to be the case.

I'm happy to go through and clean up all the workflows in this way, but would prefer to limit the scope of this PR.